### DISCUSSION TITLE
transfer github repostiry to wordpress?<title>

### DISCUSSION THREAD

**Discussion by Petscageo** [community/community#61847] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi Can I transfer my github repostiry to worpress?

**Comment by BaraaBasim** [comment]
No, it's not possible to directly import your github repository into wordpress. Wordpress is used to make dynamic and static pages and your code doesn't work just by uploading it to wordpress you would need to follow the WordPress coding standards and best practices, and ensure that your code adheres to the GNU General Public License (GPL), which is the license used for WordPress.

**Comment by garousiamir** [comment]
I suggest you to use https://appsero.com/
you can see in the middle of their homepage saying:
..............................
Love coding, but hate managing? No problem! Just push your code to Git (GitHub, Bitbucket, or Gitlab) - our intelligent tool will automatically deploy it to WordPress.org and other channels.
